routes/web.php

```diff
@@ -139,13 +139,16 @@
     $allKelas = Kelas::all();
     $comingFrom = request('comingFrom') === null ? 'none':request('comingFrom');
     $position = request('position') === null ? 0:request('position');
-    return Inertia::render('Kelas', [
-        'comingFrom' => $comingFrom,
-        'currentUser' => $user,
-        'position' => $position,
-        'userRole' => $userRole->role,
-        'allKelas' => $allKelas,
-    ]);
+    $kelasPrivilege = array(1,2,4,5);
```

Comment:
My handy dandy tips will be put these number as a note somewhere, and pass it to the next developer

Reply:
should i create comment about roles id? wkwk cause i'm tired to recheck between db and this array wkwkwk